src/components/Tab/Tabs.tsx

```diff
@@ -146,7 +151,7 @@ export const TabsComponent = forwardRef<TabsRef, TabsProps>(
             </button>
           ))}
         </div>
-        <div>
+        <div className={twMerge(tabItemContainerStyle)}>
```

Comment:
I think that you want to also merge this style with the `tabItemContainer.base`, right?

Reply:
Oh yea right, thanks!